server/service/hosts_test.go

```diff
@@ -525,11 +525,6 @@ func TestListHosts(t *testing.T) {
 	require.NoError(t, err)
 	require.Len(t, hosts, 1)
 
-	// anyone can list hosts
-	hosts, err = svc.ListHosts(test.UserContext(ctx, test.UserNoRoles), fleet.HostListOptions{})
-	require.NoError(t, err)
-	require.Len(t, hosts, 1)
```

Comment:
Shouldn't we replace this test with one that checks all but GitOps?

Reply:
This is removed because `anyone can list hosts` is not true anymore.
Tests for GitOps have been added elsewhere (see `TestGitOpsUserActions`).